geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategyV5.scala

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.locationtech.geomesa.accumulo.index
+
+import java.util.Date
+
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.accumulo.core.client.IteratorSetting
+import org.apache.accumulo.core.data.{Range => AccRange}
+import org.apache.hadoop.io.Text
+import org.geotools.factory.Hints
+import org.geotools.filter.text.ecql.ECQL
+import org.geotools.temporal.`object`.DefaultPeriod
+import org.locationtech.geomesa.accumulo._
+import org.locationtech.geomesa.accumulo.data._
+import org.locationtech.geomesa.accumulo.data.tables.{AttributeTable, AttributeTableV5, RecordTable}
+import org.locationtech.geomesa.accumulo.index.QueryHints.RichHints
+import org.locationtech.geomesa.accumulo.index.QueryPlanner._
+import org.locationtech.geomesa.accumulo.index.QueryPlanners.JoinFunction
+import org.locationtech.geomesa.accumulo.index.Strategy._
+import org.locationtech.geomesa.accumulo.iterators._
+import org.locationtech.geomesa.features.SerializationType.SerializationType
+import org.locationtech.geomesa.filter.FilterHelper._
+import org.locationtech.geomesa.filter._
+import org.locationtech.geomesa.utils.geotools.RichAttributeDescriptors.RichAttributeDescriptor
+import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
+import org.locationtech.geomesa.utils.stats.IndexCoverage._
+import org.locationtech.geomesa.utils.stats.{Cardinality, IndexCoverage}
+import org.opengis.feature.simple.SimpleFeatureType
+import org.opengis.filter._
+import org.opengis.filter.expression.{Literal, PropertyName}
+import org.opengis.filter.temporal.{After, Before, During, TEquals}
+
+import scala.collection.JavaConverters._
+
+@deprecated
+class AttributeIdxStrategyV5(val filter: QueryFilter) extends Strategy with Logging {
```

Comment:
this is the old attribute strategy, i haven't changed anything here except renamed the file.